webviz_subsurface/plugins/_group_tree/_views/__init__.py

```diff
@@ -0,0 +1,7 @@
+from ._group_tree_view import (
+    GroupTreeView,
+    GroupTreeViewElement,
+    ViewControls,
+    ViewFilters,
+    ViewOptions,
+)
```

Comment:
I am not sure if it is actually helpful to make all these classes available via `_views`. I think it is not necessary and you could empty this `__init__` file. This makes it clearer where the classes come from.